### PR TITLE
Adding Crypto.com widget to the NTP [1.16]

### PR DESCRIPTION
**Pull request by ryanml**
Uplift of: https://github.com/brave/brave-core/pull/6705, https://github.com/brave/brave-core/pull/6766
Resolves: https://github.com/brave/brave-browser/issues/12074, https://github.com/brave/brave-browser/issues/9247

Approved, please ensure that before merging: 
- [x] You have checked CI and the builds, lint, and tests all pass or are not related to your PR. 
- [x] You have tested your change on Nightly. 
- [x] The PR milestones match the branch they are landing to. 

After you merge: 
- [x] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Comment by ryanml** [comment]
Restarted MacOS due to unrelated failure

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.16.x` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-core/pull/6705#issuecomment-707566708 & https://github.com/brave/brave-core/pull/6766#issuecomment-708470110. However, @srirambv found the following issues that we'll need to get resolved in `1.16.x`. Once they're all fixed, @ryanml will create another uplift PR for `1.16.x`.

* https://github.com/brave/brave-browser/issues/12094 - Still opened, waiting on assets from Crypto.com
* https://github.com/brave/brave-browser/issues/12089 - Still opened
* https://github.com/brave/brave-browser/issues/12093 - Merged into Nightly (`1.17x.`)
* https://github.com/brave/brave-browser/issues/12092 - Merged into Nightly (`1.17x.`)
* https://github.com/brave/brave-browser/issues/12091 - Merged into Nightly (`1.17x.`)

**Pull request merged**
